static/js/data/region_codes.js

```diff
@@ -0,0 +1,5822 @@
+export default {
```

Comment:
Yes, but transformed from csv to json, and grouped by country, so that fusejs can do a fuzzy search on the state/territory names for a country